test/SparqlExpressionTest.cpp

```diff
@@ -96,8 +96,9 @@ auto checkResultsEqual = []<SingleExpressionResult A, SingleExpressionResult B>(
 auto testNaryExpression = [](auto&& makeExpression,
                              SingleExpressionResult auto&& expected,
                              SingleExpressionResult auto&&... operands) {
+  using namespace ad_utility::memory_literals;
   ad_utility::AllocatorWithLimit<Id> alloc{
-      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1000)};
+      ad_utility::makeAllocationMemoryLeftThreadsafeObject(1_kB)};
```

Comment:
Also here.

Reply:
Improving already existing tests should be a separate branch.

Reply:
Done, see [758bbbe](https://github.com/ad-freiburg/qlever/pull/1112/commits/758bbbe549e53fbf83c0fe6d30620026eef3e7a3).